### DISCUSSION TITLE
How to set the C:\Windows\Fonts\simhei.ttf font into bold in pdf myself

### DISCUSSION THREAD

**Discussion by honghh2018** [pymupdf/PyMuPDF#1812] [Q&A] [answered]
Hi @developer,
     I get a basic issue about the font form, i want to set a line character(chinese) into bold or italic, but i can exactly found the way to do it 
    had any advices for this ?

Best

**Comment by JorjMcKie** [comment, marked as answer]
To these things you must use a different font. So you must look for the bold / italic versions of `simhei.ttf` ... and they may not exist 😒.
